blockstore/src/test/java/brooklyn/location/blockstore/AbstractVolumeManagerLiveTest.java

```diff
@@ -142,6 +153,15 @@ public void testCreateVolume() throws Exception {
     // Does the attach+mount twice to ensure that cleanup worked
     @Test(groups="Live", dependsOnMethods = "testCreateVolume")
     public void testCreateAndAttachVolume() throws Exception {
+        runCreateAndAttachVolume(ReattachMode.RE_ATTACH);
+    }
+    
+    // Does the attach+mount twice to ensure that cleanup worked
+    protected enum ReattachMode {
```

Comment:
[minor] could this enum be helpful elsewhere? May be useful to move it somewhere else